Dockerfile.nightly

```diff
@@ -0,0 +1,19 @@
+FROM rockylinux/rockylinux:8
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+RUN dnf install -y https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
+
+RUN cp /etc/yum.repos.d/ondemand-web.repo /etc/yum.repos.d/ondemand-nightly-web.repo && \
+    sed -i 's/latest/nightly/g' /etc/yum.repos.d/ondemand-nightly-web.repo && \
+    sed -i 's/ondemand-web/ondemand-web-nightly/g' /etc/yum.repos.d/ondemand-nightly-web.repo
```

Comment:
@treydock is this the best way to achieve this?

Reply:
I think second sed is unnecessary but first one is correct.  I also wouldn't worry about renaming the file.  I historically have not renamed files or description when going from like latest to build/2.1 for example.